packages/vats/src/core/chain-behaviors.js

```diff
@@ -97,6 +97,21 @@ export const bridgeCoreEval = async allPowers => {
 };
 harden(bridgeCoreEval);
 
+/**
+ * @param {BootstrapPowers} powers
+ */
+export const makeOffChainBrands = async ({
```

Comment:
Since this is referenced in `SHARED_BOOTSTRAP_MANIFEST`, it should be in `basic-behaviors.js`, no? Please move it.